lib/matplotlib/figure.py

```diff
@@ -28,7 +28,7 @@
 
 from matplotlib.axes import Axes, SubplotBase, subplot_class_factory
 from matplotlib.blocking_input import BlockingMouseInput, BlockingKeyMouseInput
-from matplotlib.gridspec import GridSpec, SubplotSpec
+from matplotlib.gridspec import GridSpec, SubplotSpec  # noqa: F401
```

Comment:
I think we favor per-file-ignores in `.flake8` over `# noqa` comments to not clutter the code base.

AFAIK this was discussed at some point but is not formally written down.

Reply:
I'd rather not, as it's only one import line, not all over the file, and it's already been removed in `master`.

Reply:
What was removed on master?

Reply:
This `SubplotSpec` import: https://github.com/matplotlib/matplotlib/pull/18029/files#diff-c3ce78ddcc28c54e04ade080c149c1b1L32

Reply:
Oh. Then yeah, let's just `noqa` it.